[2020-06-01 20:21:39,718] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask1 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 20:21:39,727] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask1 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 20:21:39,728] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:21:39,728] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-01 20:21:39,728] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:21:39,743] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): multitask1> on 2020-05-30T00:00:00+00:00
[2020-06-01 20:21:39,745] {standard_task_runner.py:53} INFO - Started process 15109 to run task
[2020-06-01 20:21:39,800] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_dag_cvd.multitask1 2020-05-30T00:00:00+00:00 [running]> nineleaps
[2020-06-01 20:21:39,810] {logging_mixin.py:112} INFO - {'a': 1, 'b': 2}
[2020-06-01 20:21:39,810] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-01 20:21:39,814] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=test_dag_cvd, task_id=multitask1, execution_date=20200530T000000, start_date=20200601T145139, end_date=20200601T145139
[2020-06-01 20:21:49,710] {logging_mixin.py:112} INFO - [2020-06-01 20:21:49,710] {local_task_job.py:103} INFO - Task exited with return code 0
[2020-06-01 20:36:27,449] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask1 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 20:36:27,459] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask1 2020-05-30T00:00:00+00:00 [queued]>
[2020-06-01 20:36:27,459] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:36:27,459] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-01 20:36:27,459] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:36:27,474] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): multitask1> on 2020-05-30T00:00:00+00:00
[2020-06-01 20:36:27,476] {standard_task_runner.py:53} INFO - Started process 16801 to run task
[2020-06-01 20:36:27,531] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_dag_cvd.multitask1 2020-05-30T00:00:00+00:00 [running]> nineleaps
[2020-06-01 20:36:27,540] {logging_mixin.py:112} INFO - {'a': 1, 'b': 2}
[2020-06-01 20:36:27,540] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-01 20:36:27,543] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=test_dag_cvd, task_id=multitask1, execution_date=20200530T000000, start_date=20200601T150627, end_date=20200601T150627
[2020-06-01 20:36:37,429] {logging_mixin.py:112} INFO - [2020-06-01 20:36:37,428] {local_task_job.py:103} INFO - Task exited with return code 0
